t/e2e/accounts/accounts_test.go

```diff
@@ -225,19 +243,27 @@ func (s *AccountsTestSuite) TestSelectedAccountOnRestart() {
 	s.NoError(s.Backend.StopNode())
 
 	// make sure that account is still selected
-	selectedAccount, err = s.Backend.AccountManager().SelectedWalletAccount()
+	selectedWalletAccount, err = s.Backend.AccountManager().SelectedWalletAccount()
+	s.NoError(err)
```

Comment:
Do we want to continue the test if `SelectedWalletAccount` returns an error?
```suggestion
	s.Require().NoError(err)
```